### PR TITLE
generate the data file instead

### PR DESCRIPTION
**Pull request by alshabib**

### CODE REVIEW

File: file/file_test.go [hunk 1]
```diff
@@ -63,7 +68,31 @@ func (*fakePutClient) CloseSend() error {
 	return nil
 }
 
+func generateFile(t *testing.T) string {
+	// Create a temporary file
+	file, err := os.CreateTemp("/tmp", "data-*.txt")
```
[the remark this answers is not in the document]
Comment (alshabib) [reply file/file_test.go]: Ah good to know!

File: file/file_test.go [hunk 2]
```diff
@@ -28,6 +29,10 @@ import (
 	"google.golang.org/protobuf/testing/protocmp"
 )
 
+const (
+	data = "some really important data"
```
[the remark this answers is not in the document]
Comment (alshabib) [reply file/file_test.go:34]: done

File: file/file_test.go [hunk 3]
```diff
@@ -63,7 +69,20 @@ func (*fakePutClient) CloseSend() error {
...
+	// Write some text to the file
+	if err := os.WriteFile(fileName, []byte(data), 0644); err != nil {
+		t.Fatalf("unable to write temp file contents: %v", err)
+	}
+
+	return fileName
+}
+
 func TestPut(t *testing.T) {
+	fileName := generateFile(t, data)
```
Comment (greg-dennis) [inline file/file_test.go:85]: add 
`const data = "some really important data"`
above this line

and delete the global const

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: file/file_test.go (hunk 3 above)
Comment (alshabib) [reply file/file_test.go:82]: done

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**